### DISCUSSION TITLE
tensorRT workspace size definition

### DISCUSSION THREAD

**Discussion by ausollet** [ultralytics/yolov5#6472] [Q&A] [answered]
Hey, I was going through the export.py code and was wondering what exactly is Workspace argument since it says "Some tactics do not have sufficient workspace memory to run. Increasing workspace size may increase performance, please check verbose output" when I run the code to convert .pt weights to .engine.

Thanks

**Comment by glenn-jocher** [comment, marked as answer]
@ausollet default workspace size is 4GB. You can increase this with the `--workspace` argument, i.e. for 8GB:

```bash
python export.py --include tensorrt --workspace 8
```

**Reply by ausollet** [reply]
Thank you, that part was clear from the code, I wanted to know what exactly is this argument and what is it for.

**Reply by ausollet** [reply]
Does it refer to the max GPU resource that can be allocated?

**Reply by glenn-jocher** [reply]
@ausollet it refers the TensoRT workspace size allocated during export.
